docs/samples/README.md

```diff
@@ -166,7 +177,7 @@ When you install the `coherence-operator` you can optionally enable the followin
 1. Prometheus integration to capture metrics and display in Grafana. (Only available from Coherence 12.2.1.4.0 and above)
 
 1. Log capture which will use Fluentd to send logs to Elasticsearch where
-   they key be vied in Kiabana.
+   they key be viewed in Kiabana.
```

Comment:
I think this should be "they *can* be viewed in Kibana"

Reply:
Thanks, fixed.